packages/tools/generator-widget/generators/app/templates/pluggable/web/emptyTemplateTsFn/src/WidgetName.editorPreview.tsx.ejs

```diff
@@ -0,0 +1,13 @@
+import { ReactElement, createElement } from "react";
+import { HelloWorldSample } from "./components/HelloWorldSample";
+import { <%- name %>PreviewProps } from "../typings/<%- name %>Props";
+
+declare function require(name: string): string;
```

Comment:
We might not need it anymore as the main tsconfig from pluggable-widgets-tools adds node to the typings.